baseline.gradle

```diff
@@ -57,8 +57,8 @@ subprojects {
   pluginManager.withPlugin('com.palantir.baseline-error-prone') {
     tasks.withType(JavaCompile).configureEach {
       options.errorprone.errorproneArgs.addAll (
-          // error-prone is slow, don't run on tests and generated src
-          '-XepExcludedPaths:.*/(test|generated-src)/.*',
+          // error-prone is slow, don't run on tests/generated-src/generated
+          '-XepExcludedPaths:.*/(test|generated-src|generated)/.*',
```

Comment:
Can we use the same `generated-src` directory? Otherwise the generated classes don't show up in an IDE without reconfiguring. If not, we should discuss this on the dev list to make sure everyone knows it's going to cause trouble in IDEs.

Reply:
Actually, after running build I do see the classes show up without reconfiguring. Looks like this should be okay.

Reply:
yes this should be perfectly ok, it just needs a single refresh in the IDE for the generated classes to show up